README.md

```diff
@@ -253,7 +253,8 @@ Production deployments can be run with `VERSION=<some-docker-tag-or-semver> make
 
 Current workflow:
 1. merge from `staging` into `master`
-1. tag the desired commit on `master` with a `v#.#.#` format and push the tag
+1. "Draft a new release" on https://github.com/sillsdev/web-languageforge/releases
```

Comment:
```suggestion
1. "Draft a new release" on https://github.com/sillsdev/web-languageforge/releases with a `v#.#.#` tag format
```
This is a bit of a nit but it seems important to leave in instructions on tag format since that still applies

Reply:
No, that's a good idea, thanks!